app/controllers/rentals_controller.rb

```diff
@@ -41,9 +41,11 @@ def processing
 
   # GET /rentals/1/transform
   def transform
-    if @rental.rental_status == 'reserved'
+    if @rental.reserved? && @rental.end_date < DateTime.current
```

Comment:
Please add some time_cop checks into the controller specs.

One concern: Confirm that time zones don't cause you issues. The app is running in eastern time, I just would like to see a test confirming the validity of the check.